### PR TITLE
Import the grades_persistentcoursegrade table

### PR DESCRIPTION
**Pull request by iloveagent57**
In the interest of smaller, more frequent PRs, here's the first piece of https://openedx.atlassian.net/browse/TNL-6871. The new task isn't wired up to anything yet, that will happen in the next PR for this story. Also included here is some refactoring to reduce Luigi parameter repetition between mixin classes.

### CODE REVIEW

**Comment by mulby** [comment]
I'm still nervous about the size of this table growing significantly over time... I wonder if we couldn't come up with a simple and clean incremental solution using the events?

File: edx/analytics/tasks/common/sqoop.py [hunk 1]
```diff
@@ -23,57 +23,66 @@ def load_sqoop_cmd():
...
+        "host": "db.example.com",
+        "port": "3306",
+        "username": "exampleuser",
+        "password": "example password"
+    }
     """
     destination = luigi.Parameter(
         config_path={'section': 'database-import', 'name': 'destination'},
-        description='The directory to write the output files to.',
+        description='The directory to write the output files to.'
```
Comment (brianhw) [inline edx/analytics/tasks/common/sqoop.py]: nit:  interesting that this removes a comma, when the database description below adds a comma.  I think the comma is good, e.g. when adding a new line like 'significant'.

File: edx/analytics/tasks/common/sqoop.py [hunk 2]
```diff
@@ -23,57 +23,66 @@ def load_sqoop_cmd():
...
         default=False,
+        significant=False,
         description='Print more information while working.',
     )
-    table_name = luigi.Parameter(
-        description='The name of the table to import.',
-    )
     where = luigi.Parameter(
         default=None,
+        significant=False,
```
Comment (brianhw) [inline edx/analytics/tasks/common/sqoop.py]: Why is this not significant?   Adding such a clause should change the result, where num_mappers and verbose settings do not.

File: edx/analytics/tasks/common/sqoop.py [hunk 3]
```diff
@@ -23,57 +23,66 @@ def load_sqoop_cmd():
...
         description='A "where" clause to be passed to Sqoop.  Note that '
         'no spaces should be embedded and special characters should '
         'be escaped.  For example:  --where "id\<50". ',
     )
+
+
+class SqoopImportTask(OverwriteOutputMixin, luigi.hadoop.BaseHadoopJobTask, SqoopImportMixin):
```
Comment (brianhw) [inline edx/analytics/tasks/common/sqoop.py]: Is there a basic ordering here?  Generally we have put the mixins before the base class.

**Review by brianhw — APPROVED** [review]
LGTM.   Just a few nits.

File: edx/analytics/tasks/common/sqoop.py (hunk 2 above)
Comment (iloveagent57) [reply edx/analytics/tasks/common/sqoop.py]: I think that was copypasta.

File: edx/analytics/tasks/common/sqoop.py (hunk 3 above)
Comment (iloveagent57) [reply edx/analytics/tasks/common/sqoop.py]: Good point.

**Pull request merged**